src/components/AvatarWithIndicator.js

```diff
@@ -10,24 +10,33 @@ const propTypes = {
 
     // url for the avatar
     source: PropTypes.string.isRequired,
+
+    // avatar size
+    size: PropTypes.string,
+
 };
 
 const defaultProps = {
     isActive: false,
+    size: 'normal',
 };
 
 const AvatarWithIndicator = ({
     isActive,
     source,
+    size,
 }) => {
     const indicatorStyles = [
-        styles.statusIndicator,
+        size === 'large' ? styles.settingsStatusIndicator : styles.statusIndicator,
         isActive ? styles.statusIndicatorOnline : styles.statusIndicatorOffline,
     ];
 
     return (
-        <View style={[styles.sidebarAvatar]}>
+        <View
+            style={[size === 'large' ? styles.settingPageAvatar : styles.sidebarAvatar]}
```

Comment:
`settingPageAvatar` can we change this name to something more generic? It may have nothing to do with the "settings page".